plugins/ibm_qradar/plugin.spec.yaml

```
@@ -0,0 +1,212 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: ibm_qradar
+title: IBM Qradar
+vendor: rapid7-sacumen
+support: community
+status: []
+description: Orchestrate and automate ariel search and automate the offence management using the IBM Qradar
+version: 1.0.8
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/plugins/ibm_qradar
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://www.ibm.com
+tags:
+  - qradar
+  - ariel search
```

I would suggest the most natural tags relating to IBM Ariel database and the [AQL query language](https://www.ibm.com/docs/en/qradar-on-cloud?topic=aql-query-structure) are as follows:

```suggestion
  - ariel
  - aql
```

if else, at least change `ariel search` from space to underscore `ariel_search`.